generate.go

```diff
@@ -1,4 +1,38 @@
-//go:generate go-bindata -pkg=data -ignore=.*\.go -o data.go . ./benkeen/d3pie/ ./jiahuang/d3-timeline/ ./krisk/fuse/ ./zeroclipboard/zeroclipboard/ ./seiyria/bootstrap-slider/
-//go:generate go fmt .
+// +build generate
 
-package data
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+	pathpkg "path"
```

Comment:
Wow that's an extensive list of references to renaming.. not sure I can argue against that :laughing: 

Personally I think packages should really only be renamed if two imported package names conflict with one another.. but given all the examples you've provided to the contrary it's fine to leave it as is :)